ci-operator/step-registry/origin/e2e-ovn/OWNERS

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- danwinship
+- dcbw
+- knobunc
+- pecameron
+- squeed
+options: {}
```

Comment:
nit: no need to set `options` unless you want to fill in some subsequent setting.  Can we drop this line?

Reply:
done and did the same thing to ci-operator/step-registry/ovn/OWNERS